pkgs/development/python-modules/pyscard/default.nix

```
@@ -30,9 +24,9 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
-  nativeBuildInputs = [ swig ] ++ lib.optionals (!withApplePCSC) [ pkg-config ];
+  nativeBuildInputs = [ swig ] ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ pkg-config ];
 
-  buildInputs = if withApplePCSC then [ PCSC ] else [ pcsclite ];
+  buildInputs = lib.optional (!stdenv.hostPlatform.isDarwin) pcsclite;
```

nit:
```suggestion
  buildInputs = lib.optionals (!stdenv.hostPlatform.isDarwin) [ pcsclite ];
```
for consistency (and because `lib.optional` is a slight footgun).

---

@emilazy Will do; but could you please explain the footgun part so that I know why I'm doing it?

---

`lib.optional cond [ pkg ]` produces `[ [ pkg ] ]`, which doesn’t behave like you might expect. Since `lib.optionals` also scales better when you add more, I (and others) personally prefer using `lib.optionals` in all cases.